src/core/tokens/_core-typography.scss

```diff
@@ -0,0 +1,50 @@
+// CORE TOKENS: TYPOGRAPHY
+// This partial Defines Global Typography tokens
+
+// FONTS SCALES
+// - Font Size scale
+$font-size-10: $font-reg-size-14;
+$font-size-20: $font-reg-size-14;
+$font-size-30: $font-size-base;
+$font-size-40: $font-reg-size-18;
+$font-size-50: $font-reg-size-24;
+$font-size-60: $font-reg-size-32;
+
+// - LINE HEIGHT SCALE
+$font-line-height-10: $font-reg-line-height-18;
+$font-line-height-20: $font-reg-line-height-24;
+$font-line-height-30: $font-reg-line-height-24;
+$font-line-height-40: $font-reg-line-height-24;
+$font-line-height-50: $font-reg-line-height-32;
+$font-line-height-60: $font-reg-line-height-52;
+
+// TYPOGRAPHY TOKENS
+// - Font sizes (meaningful units)
+$font-size-h1: var(--font-size-60);
```

Comment:
we want to expose as little as variables as we can - `--font-size-##` will be available for everyone to use and it doesn't imply the developer when to use it

Reply:
this is the case right now. 
i did not change it, just wrote it in a token that is generate from a map using a mixin.
the output is the same.

And i agree , this should not be exposed.
Yet in my scope i did not change the output besides removing duplication and changing sort order in some cases.